lib/internal/url.js

```diff
@@ -219,6 +219,14 @@ class URL {
     if (base !== undefined && !(base instanceof URL))
       base = new URL(String(base));
     parse(this, input, base);
+
+    Object.defineProperty(this, Symbol.toStringTag, {
+      value: 'URL',
+      writable: false,
+      enumerable: false,
+      configurable: true
+    });
```

Comment:
Can't it still be defined on the prototype and just make that decision as a getter when invoked? I'm not sure it's great to add another property on every URL object just for this?